multicore_runtime/legacy_e2e_support.py

```diff
@@ -54,7 +54,7 @@ def clean_up(self):
 
 
 def load_legacy_scripts_into_handlers(handlers):
-  """Preloads legacy CGI scripts.
+  """Preloads legacy CGI scripts. Static file handlers are not supported.
```

Comment:
To reflect that static files are unsupported? No, legacy e2e tests are minimal but separate from the normal tests and all tests still pass.